composer.json

```
@@ -18,7 +18,7 @@
 		"optimize-autoloader": true,
 		"classmap-authoritative": true,
 		"platform": {
-			"php": "7.4"
+			"php": "8.0"
```

Server requires 8.0.2


```suggestion
			"php": "8.0.2"
```